modules/mod_asgi/autogen.sh

```
@@ -0,0 +1,30 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License along with
+# Hive Viriatum Modules. If not, see <http://www.apache.org/licenses/>.
+
+# __author__    = João Magalhães <joamag@hive.pt>
+# __copyright__ = Copyright (c) 2008-2026 Hive Solutions Lda.
+# __license__   = Apache License, Version 2.0
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)